code/__DEFINES/preferences.dm

```diff
@@ -140,3 +140,10 @@
 
 /// The key used for sprite accessories that should never actually be applied to the player.
 #define SPRITE_ACCESSORY_NONE "None"
+
+// SKYRAT EDIT ADDITION BEGIN - Antag Opt In
```

Comment:
We have code/__DEFINES/~skyrat_defines/preferences.dm for this, so you don't need to non-modularly edit this. :)